test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -1038,10 +1038,10 @@ def addSuccess(self, test):
     def _cleanup(self):
         self.logger.info('Cleaning up temporary files')
         du = DshUtils()
-        hosts = self.param_dict['moms']
+        hosts = list(self.param_dict['moms'])
         for server in self.param_dict['servers']:
             if server not in self.param_dict['moms']:
-                hosts.update(self.param_dict['servers'])
+                hosts.extend(self.param_dict['servers'])
```

Comment:
Python's `set` doesn't have extend, so `.update` was right, but there is bug in old (as well as) new like is if `server` not in `moms` set then we add whole `servers` set to `hosts`, where we need to add only `server` in `hosts`

So this lines should be like:
```python
hosts = self.param_dict['moms']
for server in self.param_dict['servers']:
    if server not in hosts:
        hosts.update(server)
```